en/airframes_multicopter/qav-r-5-kiss-esc-racer.md

```diff
@@ -0,0 +1,166 @@
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_135817_s.jpg)
+
+For this method you cut the big end with the hole off the zip tie, put the rest together with the antenna cable through a long heat shrink and mount this to your frame spacers using a bigger but shorter heat shrink.
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_140814_s.jpg)
+
+### ESC signal
+
+For the ESC signals I followed the [hardware documentation of the Pixracer](https://docs.px4.io/en/flight_controller/pixracer.html) and the [Quadrotor x configuration](../airframes/airframe_reference.html#quadrotor-x) motor numbering scheme. As we have no ground or positive BEC voltage connections we connect our `PWM` ESC signal cables each to its topmost pins of the corresponding output connector.
+
+### GPS / External Magnetometer
+
+I took the GPS cable which fits the connector of the used GPS and came with the Pixracer set. Sadly the pin assignment was completely wrong and I rewired the connector again using tweezers according to the [3DR pixhawk mini user manual](../flight_controller/pixhawk_mini.md#connector-pin-assignments-pin-outs) GPS port.
+
+#### Pixracer GPS/I2C Port
+| Pin  | Assignment |
+| ---- | ---------- |
+| 1    | GND        |
+| 2    | SDA        |
+| 3    | SCL        |
+| 4    | RX         |
+| 5    | TX         |
+| 6    | +5V        |
+
+#### M8N 3DR pixhawk mini GPS Connector
+| Pin     | Assignment | Connect to Pixracer Pin |
+| ------- | ---------- | ----------------------- |
+| 1 (red) | SCL        | 3                       |
+| 2       | SDA        | 2                       |
+| 3       | VCC 5V     | 6                       |
+| 4       | RX         | 5                       |
+| 5       | TX         | 4                       |
+| 6       | GND        | 1                       |
+
+I mounted the GPS using double sided tape to the far back of the top plate of the frame. This covers the power cord and FPV antenna holes. If the distance to interference is big enough needs to be tested.
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/IMG_20170725_155515_s.jpg)
+
+## Software Configuration
+
+> **Warning**
+> Always make sure to have either battery or propellers pyhsically removed from your vehicle during any initial configuration. Better safe than sorry!
+
+For general configuration instruction please refer to the [step-by-step guide](https://docs.px4.io/en/config/).
+
+For this build I pulled the newest PX4 master because it supports the "FMU as task" improvements I'll exaplain later and flashed it to the Pixracer. Then I calibrated the sensors and while doing this had to set the external magnetometer mounting rotation to `YAW_270` even though I mounted it the oppsoite way (with the printed arrow facing upwards but pointing backwards, see pictures). 
```

Comment:
Yes I wanted to but how?
I'm looking for it. There must be some linking like HTML anchors....